app/views/hub/metrics/index.html.erb

```diff
@@ -21,79 +21,172 @@
-        <tr class="index-table__row org">
+        <tr class="index-table__row org" data-js-count="<%= @report.active_sla_clients[vita_partner.id] %>">
           <td class="index-table__cell"><%= vita_partner.name %></td>
           <td class="index-table__cell attention-needed-breach"  data-js-count=<%= @report.attention_needed_breaches[vita_partner.id] %>>
             <%= @report.attention_needed_breaches[vita_partner.id]%>
           </td>
+          <td class="index-table__cell attention-needed-breach-percentage">
+            <%= @report.active_sla_clients_count[vita_partner.id] %>
+
+          </td>
           <td class="index-table__cell communication-breach" data-js-count=<%= @report.communication_breaches[vita_partner.id] %>>
             <%= @report.communication_breaches[vita_partner.id]%>
           </td>
+
+          <td class="index-table__cell communication-breach-percentage">
+            <%= @report.active_sla_clients_count[vita_partner.id] %>
+
+          </td>
           <td class="index-table__cell interaction-breach" data-js-count=<%= @report.interaction_breaches[vita_partner.id] %>>
             <%= @report.interaction_breaches[vita_partner.id]%>
           </td>
+          <td class="index-table__cell interaction-breach-percentage">
+            <%= @report.active_sla_clients_count[vita_partner.id] %>
+          </td>
         </tr>
         <% if vita_partner.child_sites.present? %>
           <!--           Add a row for the organization to track org-level breaches.-->
-          <tr class="index-table__row site">
+          <tr class="index-table__row site" data-js-count=<%= @report.active_sla_clients[vita_partner.id] %>>
             <td class="index-table__cell" style="padding-left: 60px; font-style: italic;"><%= vita_partner.name %></td>
             <td class="index-table__cell attention-needed-breach"  data-js-count=<%= @report.attention_needed_breaches[vita_partner.id] %>>
               <%= @report.attention_needed_breaches[vita_partner.id] %>
             </td>
+            <td class="index-table__cell attention-needed-breach-percentage">
+              <%= @report.active_sla_clients_count[vita_partner.id] %>
+
+            </td>
             <td class="index-table__cell communication-breach" data-js-count=<%= @report.communication_breaches[vita_partner.id] %>>
               <%= @report.communication_breaches[vita_partner.id]%>
             </td>
+            <td class="index-table__cell communication-breach-percentage">
+              <%= @report.active_sla_clients_count[vita_partner.id] %>
+
+            </td>
             <td class="index-table__cell interaction-breach" data-js-count=<%= @report.interaction_breaches[vita_partner.id] %>>
               <%= @report.interaction_breaches[vita_partner.id] %>
             </td>
+            <td class="index-table__cell interaction-breach-percentage">
+              <%= @report.active_sla_clients_count[vita_partner.id] %>
```

Comment:
I see, we start out with the denominator as the text in these percentage cells

Reply:
Yeah, we don't really need to... I'm not sure if it makes more sense to just not show anything in the case you don't have JS turned on or leave it blank.